lib/signaling/v2/peerconnection.js

```diff
@@ -620,6 +646,16 @@ class PeerConnectionV2 extends StateMachine {
       if (!this._negotiationRole) {
         this._negotiationRole = 'offerer';
       }
+
+      if (this._shouldApplySimulcast) {
+        // cache local description before applying simulcast
```

Comment:
@syerrapragada Nitpick: No need for this comment since it is fairly obvious from the code.